backend/package.json

```diff
@@ -14,11 +14,18 @@
   "dependencies": {
     "@filecoin-station/deal-observer-db": "^1.0.0",
     "@influxdata/influxdb-client": "^1.35.0",
+    "@ipld/dag-cbor": "^9.2.2",
+    "@ipld/dag-json": "^10.2.3",
+    "@ipld/schema": "^6.0.5",
     "@sentry/node": "^8.48.0",
+    "@sinclair/typebox": "^0.34.14",
     "@types/slug": "^5.0.9",
     "debug": "^4.4.0",
     "ethers": "^6.13.5",
+    "multiformats": "^13.3.1",
     "pg": "^8.13.1",
-    "slug": "^10.0.0"
+    "request": "^2.88.2",
```

Comment:
The request module is not maintained anymore. Please use the built-in Fetch API instead.

Reply:
@NikolasHaimerl please remove this line (the unused dependency).